crates/stackable-operator/src/builder/pod/probe.rs

```diff
@@ -53,7 +53,7 @@ pub enum Error {
     PeriodIsZero {},
 }
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
Is there a case where `ProbeBuilder` needs to be `Clone`?
It should be a single use thing that is consumed at the end and returns the _built_ thing (`Probe`).

I can understand `ProbeAction` being `Clone` though.

Reply:
I'm inclined to approve this now instead of make you wait.
We can also work on builder style guidelines later.